package.json

```diff
@@ -35,6 +35,7 @@
     "jest-serializer-vue": "2.0.2",
     "p-map": "^4.0.0",
     "prettier": "^2.8.3",
+    "svg-path-parse": "^1.1.3",
```

Comment:
It should be noted that this package has not been updated for eight years.
Alternative (2y no update https://www.npmjs.com/package/svgpath) also provides scale function

Reply:
Ha, I just recommended the same package... without having seen your comment. :sweat_smile: